### PR TITLE
ADD. 1. update user Info, show more information 2. modify Repos info with description and update timestamp 3. add another column Projects and show projects as list like Repos

### PR DESCRIPTION
**Pull request by ritata**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import "./styles.css";
+import ConvertToDate from "./Util.js";
```
Comment (chucklien) [inline src/App.js]: in react, we usually us `Captital` as component, and others use lower case.
so recommend use `convertToDate` and `util.js`

File: src/Util.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function ConvertToDate(props) {
+  const date = new Date(props.time).toLocaleDateString();
+  return date;
+}
```
Comment (chucklien) [inline src/Util.js]: I thought it's util function to get local date string, not a  react-component.
as i know, for react component, we need return a react.node not a pure string, that's what I mention above, use lower case for `convertToDate`

File: src/App.js [hunk 3]
```diff
@@ -10,31 +11,74 @@ function Profile(props) {
...
+        {projects.description}
+        <br />
+        Update on <ConvertToDate time={projects.updateTimestamp} />
+      </div>
+      <hr />
+    </li>
+  ));
+  return <ul>{listProjects}</ul>;
+}
+
```
Comment (chucklien) [inline src/App.js:44]: please check this two component `FavoriteRepos` and `MyProjects`, don't you think they look familiar?

Comment (chucklien) [reply src/App.js:44]: may be we could use `RepoList` and take props `data`
```javascript
<RepoList data={favoriteList} />
<ReposList data={projects} />
```
so we could re-use `ReposList`

**Review by chucklien — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (chucklien) [reply src/App.js]: if you take `ConvertToDate` as a react component, you could ignore this.

File: src/Util.js (hunk 2 above)
Comment (chucklien) [reply src/Util.js]: since this is used to get locale date string, you could treat this as pure js function.

File: src/App.js (hunk 3 above)
Comment (chucklien) [reply src/App.js:44]: or 
```javascript
const FavList = <RepoList data={favoriteList} />;
const MyProject = <RepoList data={projects} />;
```
the purpose is used to DRY, maybe in the future you think they are not the same data structure, you could split easily

File: src/App.js (hunk 1 above)
Comment (ritata) [reply src/App.js]: since I will modify it to pure js function, so I will use `convertToDate` and `util.js`.

File: src/Util.js (hunk 2 above)
Comment (ritata) [reply src/Util.js]: ok, I will modify it to pure JS function.

File: src/App.js (hunk 3 above)
Comment (ritata) [reply src/App.js:44]: thanks, I will refactor it.
use common component when display content and remove duplicate component.

**Review by chucklien — APPROVED** [review]

**Pull request merged**